tests/test_integration.py

```diff
@@ -163,6 +163,7 @@ def test_sched_a_fulltext_trigger(self):
         )
         db.session.commit()
         manage.update_aggregates()
+        manage.refresh_itemized()
```

Comment:
Oh I know why - most of these tests are against schedule A or B records, which expect the queue processing to have been done.  A couple don't though, so I will double check and remove as I can.